sxw_resource.c

```diff
@@ -150,14 +119,14 @@ void _sxw_update_resource(void) {
 	_sxw_update_root_tables(sizes);
 
 	/* Assign transpiration (resource availability) to each STEPPE functional group */
-	_transp_contribution_by_group(_resource_cur);
+	_transp_contribution_by_group(SXWResources->_resource_cur);
 
         /* Scale transpiration resources by a constant, bvt, to convert resources
          * (cm) to biomass that can be supported by those resources (g/cm) */
 	ForEachGroup(g)
 	{
 //printf("for groupName= %smresource_cur prior to multiplication: %f\n",RGroup[g]->name, _resource_cur[g]);
-		_resource_cur[g] = _resource_cur[g] * _bvt;
+		SXWResources->_resource_cur[g] = SXWResources->_resource_cur[g] * SXWResources->_bvt;
 //printf("for groupName= %s, resource_cur post multiplication: %f\n\n",Rgroup[g]->name, _resource_cur[g]);
 	}
 /* _print_debuginfo(); */
```

Comment:
This line can probably safely be removed.

Reply:
Resolved by 103ca00.